FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/response/api/LinkMessageResponse.kt

```diff
@@ -12,7 +12,7 @@ import com.google.gson.annotations.SerializedName
 import kotlinx.parcelize.Parcelize
 
 /**
- * This class describes a response from the https://flowcrypt.com/api/link/message API.
+ * This class describes a response from the https://flowcrypt.com/shared-tenant-fes/link/message API.
  *
  *
  * `POST /initial/confirm
```

Comment:
This class is probably to be removed too? Since responses from FES probably use a different Response.